next/components/organisms/NavBar/AlertBanner.tsx

```diff
@@ -0,0 +1,59 @@
+import { AlertIcon, CrossIcon } from '@assets/images'
+import Markdown from '@components/atoms/Markdown'
+import Button from '@components/forms/simple-components/Button'
+import SectionContainer from '@components/ui/SectionContainer/SectionContainer'
+import { useGeneralContext } from '@utils/generalContext'
+import cx from 'classnames'
+import React, { forwardRef, useEffect, useState } from 'react'
+import { useLocalStorage } from 'usehooks-ts'
+
+const AlertBanner = forwardRef<HTMLDivElement>((props, forwardedRef) => {
+  const { alert } = useGeneralContext()
+  const { text, updatedAt } = alert?.data?.attributes ?? {}
+  const storageKey = `bratislava-sk-dismissible-alert-timestamp_${updatedAt}`
+
+  const [showAlert, setShowAlert] = useState(false)
+  const [wasShown, setWasShown] = useLocalStorage(storageKey, false)
+
+  useEffect(() => {
```

Comment:
This is an antipattern:
https://twitter.com/housecor/status/1584883279874117633